h2o-py/tests/testdir_algos/xgboost/pyunit_xgboost_zero_nan_handling_native_comparison.py

```diff
@@ -43,15 +59,15 @@ def xgboost_categorical_zero_nan_handling_sparse_vs_native():
     prediction_frame = h2o.H2OFrame(prediction_frame)
     prediction_frame['col1'] = prediction_frame['col1'].asnumeric()
 
-    sparse_trained_model = H2OXGBoostEstimator(training_frame=training_frame, learn_rate=1, max_depth=2,
-                                               booster='gbtree', seed=1, ntrees=1, dmatrix_type='sparse', stopping_rounds=0)
+    sparse_trained_model = H2OXGBoostEstimator(**h2oParamsS)
     sparse_trained_model.train(x=['col1'], y='response', training_frame=training_frame)
     sparse_based_prediction = sparse_trained_model.predict(prediction_frame['col1'])
-    print(sparse_based_prediction)
 
     #Prediction should be equal. In both cases, 0 and NaN should be treated the same (default direction for NaN)
-    assert sparse_based_prediction['predict'][0,0] == native_prediction[0].item()
-    assert sparse_based_prediction['predict'][1,0] == native_prediction[1].item()
+    print(native_prediction)
+    print(sparse_based_prediction)
+    assert sparse_based_prediction['predict'][0,0] == sparse_based_prediction['predict'][1,0]
```

Comment:
I test here that the two first predictions are the same (predicting NaN and Zero). They are indeed the same, but Native XGBoost will give a mean of the two values - 25, however H2O will give better result - 30. As ETA is set exactly to 1, it should immediately learn the value 30. In case of native XGBoost, that does not happen. Maybe this is a parameter issue. Spent quite some time trying to figure out what's wrong with parameters, found no difference.

However, it treats the both values the same way, which was the purpose of this test and the asserts test that.